SOURCES/xapi-1.249.3-update-db-tunnel-protocol-from-other_config.XCP-ng.patch

```diff
@@ -0,0 +1,39 @@
+diff --git i/ocaml/xapi/xapi_db_upgrade.ml w/ocaml/xapi/xapi_db_upgrade.ml
```

Comment:
Since we don't have a github issue that describes what we solve, we need a complete description of the context of that patch here, especially since each new release will have us wonder if we still need that patch or not (spoiler: we probably will forever, until we maybe decide that we accept possible loss of tunnel configuration upon upgrade from old instances).

Improving the commit message to explain the *why* of the commit would be good too.

Reply:
IMHO we should drop it for 8.3.

Reply:
The updated description does not describe the *why* of the commit:
* we used to put protocol information in other_config in versions before 8.2
* in 8.2 the sdn patch was reworked and caused a database change: the protocol information is now in a dedicated field
* this patch is required to move the setting into the right field for existing tunnels

+ mention that this patch is ideally required as long as we support upgrade from a version lower than 8.2, unless we decide that a few users will need to recreate their tunnels after the upgrade.

Reply:
s/previous de/prior to/ ?
s/exist/exists/
s/Now the/Now that the/

Reply:
Indeed ^^'